docs/index.md

```diff
@@ -47,13 +76,24 @@ so we must use eval-rst here
    openff.bespokefit
 :::
 
+:::{toctree}
+---
+maxdepth: 2
+caption: "Command Line Reference"
+glob: True
+---
+
+ref/cli
 
+:::
 
-<!-- This matches the styling of a toctree caption as of Sphinx 4.2.0 -->
-<div class="toctree-wrapper"><p class="caption" role="heading"><span class="caption-text">
-Indices
-</span></p></div>
+:::{toctree}
+---
+maxdepth: 2
+caption: "Developer Guide"
+glob: True
+---
 
-* [](genindex)
-* [](modindex)
-* [](search)
+developers/schemas
+
+:::
```

Comment:
I resisted writing up Pydantic schemas for a while until I decided that a basic understanding is pretty essential to understanding the Python API - jargon like fields and models and schemas are all over the API reference, and thinking of some python types as just data (as opposed to stateful behaviour) isn't necessarily totally obvious to Pythonistas. So I'd suggest either moving the schemas chapter back into the user's guide. I'm not sure the schemas chapter is really enough details for developers of bespoke-fit, who would be better served by the Pydantic docs.

Reply:
Hmm that's a good point. We may want to go over the API docs themselves to try and make them more clear. Terms like *field* (sometimes called *attributes* or *properties*), *model*, and *schema* are I think quite common terms in programming, and are agnostic of `pydantic`, but we may want to try and make their usage a little more consistent.

Because this package is predominately industry facing, I think in general most users will be more inclined to use the CLI as that is how they typically seem to interact with their tools. For those who do write python, I imagine they will mostly be copying the code examples and maybe making slight edits based on the text in the main user docs rather than reading the API docs, so I think it's fine for the API docs to be a little more technical / jargony 🙂